eng/pipelines/handlers.yml

```diff
@@ -211,9 +211,10 @@ stages:
             - ${{ if eq(BuildPlatform.name, 'macos') }}:
               - script: echo '##vso[task.setvariable variable=JI_JAVA_HOME]$(JAVA_HOME_11_X64)'
                 displayName: set JI_JAVA_HOME
-            - pwsh: ./eng/package.ps1 -configuration Release
+            - script: dotnet tool restore
+              displayName: install dotnet tools
+            - script: dotnet cake --configuration=Release
```

Comment:
You might as well delete `package.ps1`, too? Nothing else uses it?

Reply:
Yea that's the plan if I get the packing working :-) 

I had mainly restored it from your branch so I could look at it while building out the cake target